src/main/java/core/basesyntax/ColorSupplier.java

```diff
@@ -0,0 +1,20 @@
+package core.basesyntax;
+
+import java.util.Random;
+
+public class ColorSupplier {
+    private String[] colors = {"Blue", "Green", "Yellow", "Red", "Black", "Brown", "Violet"};
```

Comment:
let's add static and final keyword for array

Reply:
done